org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_memberships.scss

```diff
@@ -460,3 +460,44 @@
     margin-left: -20px;
   }
 }
+
+.page-civicrm-member-search {
+  .CRM_Member_Form_Search {
+    .select2-container {
+      max-width: 230px;
+    }
+
+    .crm-accordion-body {
+      padding: 20px !important;
+
+      tr {
+        &:first-child {
+          td {
+            padding-top: 0;
+          }
+        }
+      }
+
+      td {
+        padding-left: 0;
+      }
+
+      a.crm-clear-link {
+        position: relative;
+        top: -2px;
+      }
+
+      #_qf_Search_refresh {
+        color: #fff;
+        background: $brand-primary;
+        border-color: $brand-primary;
+        text-shadow: none;
+        font-size: $font-size-base;
+        font-family: $font-family-base;
+        text-transform: uppercase;
+        padding: 8px 12px;
+        border-radius: 2px;
```

Comment:
Can u add 
-moz-border-radius
-webkit-border-radius 
for browsers compatibility ?